test/factories/audio_version_factory.rb

```diff
@@ -15,7 +15,8 @@
     after(:create) do |audio_version, evaluator|
       c = evaluator.audio_files_count
       c = (rand(5) + 1) if (evaluator.audio_files_count == 0)
-      FactoryGirl.create_list(:audio_file, c, audio_version: audio_version)
+      c = 0 if evaluator.audio_files_count < 0
```

Comment:
So now < 0 means 0, and 0 means > 0?

Reply:
just checked, no place in the code is using the current 0 value, so getting rid of the random, and simplifying this.